src/thumbnailer_near_lossless.cc

```diff
@@ -174,7 +181,7 @@ Thumbnailer::Status Thumbnailer::NearLosslessEqual(WebPData* const webp_data) {
     }
   }
 
-  std::cout << "Final near-lossless pre-processing value: " << final_near_ll
+  std::cerr << "Final near-lossless pre-processing value: " << final_near_ll
```

Comment:
This is not an error so why do you think it's better to default to cerr instead of cout? In any case this kind of debug logging from within a function should be turned off completely by default, an a verbose flag could be added to turn it on. But that can be in another PR.

Reply:
I tried a `verbose` option before removing all `EqualPSNR` debugs. I think the output is minimal enough for now.